StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsSessionManifest.swift

```diff
@@ -55,6 +55,14 @@ struct FinancialConnectionsSessionManifest: Decodable {
         case unparsable
     }
 
+    struct DisplayText: Decodable {
+        let successPane: SuccessPane?
+
+        struct SuccessPane: Decodable {
+            let subCaption: String?
+        }
+    }
+
```

Comment:
[Similar types from web](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/47f4aa39/src/linkedAccounts/shared/text/index.ts#L116-L123) - I made everything nullable, however, because this code is essentially returned as a [dictionary](https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/text/command/generate_text_for_success.rb#L39-43) from backend and we have to be extra careful in Native with non-null types